web/app/js/components/AddResources.jsx

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+import { incompleteMeshMessage } from './util/CopyUtils.jsx';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class AddResources extends React.Component {
```

Comment:
Should this be renamed to something more generic like Resource detail banner or something similar? Also we may be able to use something like this for other messages pertaining to the resource detail page. I am not sure what that would be but I can see a use were we can pass an arbitrary message to this component and then display it based on some condition. I may be overengineering and it totally doesn't have to be in this branch.

Reply:
It definitely can be a banner where we display other messages! I think we should not make this generic more until it solves an actual DRY problem though :)